README.md

```diff
@@ -159,6 +165,31 @@ tcl.queue( 'incr x', function ( err, result ) {
 ```
 
 
+### jsFunc( cmd, callback )
+
+Instructs the Tcl interpreter to create a new command: cmd.
+When this new command is encountered at any time during the life of the interpreter,
+the specified callback will be invoked with whatever parameters were supplied to cmd.
+There is currently not a command for deleting commands from the interpreter.
+
+#### Parameters
+
+| Name     | Type   | Description               |
+|----------|--------|---------------------------|
+| cmd      | String | New Tcl command to create |
+| callback | [Callback](http://nukedzn.github.io/node-tcl/docs/Tcl.html#~cmdCallback) | Callback method to handle the command |
+
+#### Example
+
+``` js
+tcl.jsFunc( 'myNewCmd', function ( arg1, arg2 ) {
```

Comment:
Could we use `tcl.proc( cmd, callback )` signature for this? It's what we've discussed in #8.

Reply:
Assuming you just (and only) mean to rename the function name.  That's no problem.